controllers/feeds/getFeedChat.js

```diff
@@ -234,10 +236,12 @@ const getFeedChatService = async (req, res) => {
         constantActor,
         isAnonym,
         isOwnPost,
+        isOwnSignedPost,
+        isOwnAnonymousPost,
         message,
         actor,
         showToast = false
-      } = getDetail(req, b, myAnonymousId.user_id);
+      } = getDetail(req, b, req?.userId, myAnonymousId?.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `getDetail` now includes two new arguments: `req?.userId` and `myAnonymousId?.user_id`. Also, two new variables `isOwnSignedPost` and `isOwnAnonymousPost` are now being destructured from the returned object. These changes seem to be consistent with the PR's purpose of distinguishing between anonymous and signed posts.
```diff
- } = getDetail(req, b, myAnonymousId.user_id);
+ } = getDetail(req, b, req?.userId, myAnonymousId?.user_id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->